brainscore_language/benchmarks/syntaxgym/test_suites.py

```diff
@@ -0,0 +1,108 @@
+suite12 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/mvrr_mod.json'
+suite13 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/nn-nv-rpl.json'
+suite14 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/npi_orc_any.json'
+suite15 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/npi_orc_ever.json'
+suite16 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/npi_src_any.json'
+suite17 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/npi_src_ever.json'
+suite18 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/npz_ambig.json'
+suite19 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/npz_ambig_mod.json'
+suite20 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/npz_obj.json'
+suite21 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/npz_obj_mod.json'
+suite22 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/number_orc.json'
+suite23 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/number_prep.json'
+suite24 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/number_src.json'
+suite25 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/reflexive_orc_fem.json'
+suite26 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/reflexive_orc_masc.json'
+suite27 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/reflexive_prep_fem.json'
+suite28 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/reflexive_prep_masc.json'
+suite29 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/reflexive_src_fem.json'
+suite30 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/reflexive_src_masc.json'
+suite31 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/subordination.json'
+suite32 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/subordination_orc-orc.json'
+suite33 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/subordination_pp-pp.json'
+suite34 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/subordination_src-src.json'
+
+test_suite_dict = {name1: suite1,
```

Comment:
I think it would be more readable to just write it out directly, i.e.
```
test_suite_dict = {
    'center_embed': 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/center_embed.json',
    'center_embed_mod': 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/center_embed_mod.json',
    ...
```
etc.

Looking at this, couldn't you even just define the names, i.e. 
```
test_suite_names = ['center_embed', 'center_embed_mod', ...]
```
etc and then build the urls dynamically, i.e.
```
test_suite_dict = {name: f'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/{name}.json'}
```